### PR TITLE
Fix Binance selectors

### LINKED ISSUE #1303: `Binance` ‧ `Terms of Service` ‧ not tracked anymore
### No version of the `Terms of Service` of service `Binance` is recorded anymore since 23 January 2024 at 12:31:55 UTC

The source document has been recorded in a snapshot, but  no version can be [extracted](https://docs.opentermsarchive.org/#tracking-terms).
After correction, it might still be possible to recover the missed versions.

### What went wrong

- The provided selector "main" has no match in the web page at 'https://www.binance.com/en/terms'

### How to resume tracking

First of all, check if the source documents are accessible through a web browser:

- [ ] [https://www.binance.com/en/terms](https://www.binance.com/en/terms)

#### If the source documents are accessible through a web browser

[Edit the declaration](https://contribute.opentermsarchive.org/en/service?json=%7B%22name%22%3A%22Binance%22%2C%22documents%22%3A%7B%22Terms+of+Service%22%3A%7B%22fetch%22%3A%22https%3A%2F%2Fwww.binance.com%2Fen%2Fterms%22%2C%22select%22%3A%5B%22main%22%5D%2C%22executeClientScripts%22%3Atrue%7D%7D%7D&destination=OpenTermsArchive%2Fcontrib-declarations&step=2):
- Try updating the selectors.
- Try switching client scripts on with expert mode.

#### If the source documents are not accessible anymore

- If the source documents have moved, find their new location and [update it](https://contribute.opentermsarchive.org/en/service?json=%7B%22name%22%3A%22Binance%22%2C%22documents%22%3A%7B%22Terms+of+Service%22%3A%7B%22fetch%22%3A%22https%3A%2F%2Fwww.binance.com%2Fen%2Fterms%22%2C%22select%22%3A%5B%22main%22%5D%2C%22executeClientScripts%22%3Atrue%7D%7D%7D&destination=OpenTermsArchive%2Fcontrib-declarations&step=2).
- If these terms have been removed, move them from the declaration to its [history file](https://docs.opentermsarchive.org/contributing-terms/#service-history), using `2024-01-23T12:31:55Z` as the `validUntil` value.
- If the service has closed, move the entire contents of the declaration to its [history file](https://docs.opentermsarchive.org/contributing-terms/#service-history), using `2024-01-23T12:31:55Z` as the `validUntil` value.

#### If none of the above works

If the source documents are accessible in a browser but fetching them always fails from the Open Terms Archive server, this is most likely because the service provider has blocked the Open Terms Archive robots from accessing its content. In this case, updating the declaration will not enable resuming tracking. Only an agreement with the service provider, an engine upgrade, or some technical workarounds provided by the administrator of this collection’s server might resume tracking.

### References

- [Latest declaration](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/declarations/Binance.json)
- [Latest version](https://github.com/OpenTermsArchive/contrib-versions/blob/main/Binance/Terms%20of%20Service.md)

### PR DESCRIPTION
**Pull request by Kissaki**
Fixes #1303, #1391

### CODE REVIEW

File: declarations/Binance.history.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "Privacy Policy": {
+    "fetch": "https://www.binance.com/en/privacy",
+    "select": [
+      "main"
+    ],
+    "executeClientScripts": true,
+    "validUntil": "2024-03-12T00:32:36.000Z"
+  }
+}
```
Comment (clementbiron) [inline declarations/Binance.history.json:18]: This is not a well formated history file ;)
```suggestion
{
  "Terms of Service": [
    {
      "fetch": "https://www.binance.com/en/terms",
      "select": [
        "main"
      ],
      "executeClientScripts": true,
      "validUntil": "2024-01-23T12:31:55.000Z"
    }
  ],
  "Privacy Policy": [
    {
      "fetch": "https://www.binance.com/en/privacy",
      "select": [
        "main"
      ],
      "executeClientScripts": true,
      "validUntil": "2024-03-12T00:32:36.000Z"
    }
  ]
}
```

**Review by clementbiron — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!

File: declarations/Binance.history.json (hunk 1 above)
Comment (Kissaki) [reply declarations/Binance.history.json:18]: looks like a copy+paste error when copying

**Review by clementbiron — APPROVED** [review]
🚀

**Pull request merged**